contracts/base/ATM/ATMToken.sol

```diff
@@ -0,0 +1,153 @@
+pragma solidity ^0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20Mintable.sol";
+import "@openzeppelin/contracts/lifecycle/Pausable.sol";
+import "./ATMTokenInterface.sol";
+
+/**
+  *  @title ATM Token for Teller DAO
+  *
+  *  @author develop@teller.finance
+ */
+
+contract ATMToken is ATMTokenInterface, ERC20Detailed, Pausable, ERC20Mintable {
+    /**
+      *  @notice ATMToken implements an ERC20 token with a supply cap and a vesting scheduling
+     */
+
+    /* State Variables */
+    uint256 private _cap;
+
+    /* Structs */
+    struct VestingTokens {
+        address beneficiary;
+        uint256 amount;
+        uint256 deadline;
+    }
+
+    /* Mappings */
+    mapping (address => VestingTokens) private _vestingBalances;
+
+    /* Constructor */
+    constructor (
+        uint256 cap
+    ) public
+      ERC20Detailed("ATMToken", "ATM", 18)
```

Comment:
Please, add these 3 params in the constructor.

Reply:
Added